Makefile

```diff
@@ -8,36 +8,36 @@ PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
 BIN_DIR := $(GOPATH)/bin				   
 BINARY ?= mobile-security-service
+RELEASE_TAG = 0.0.1
```

Comment:
This needs to be `RELEASE_TAG ?= $(CIRLE_TAG)` so when its run by Circle that the release tag is the tag that is on the git commit. Process for making a release is to tag the git commit and push it, CircleCI will then pick that up and build and push the image.

Reply:
Done